development/docker/CentOS7.Dockerfile

```diff
@@ -22,9 +22,9 @@ RUN yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.n
       xorg-x11-server-Xvfb && \
     # Install static analysis dependencies
     pip3 install \
-      flake8==2.5.4 \
-      pep8==1.6.2 \
-      pyflakes==1.3.0 \
+      flake8==3.7.9 \
```

Comment:
If this is arbitrarily "up to date" rather than a specific version then either just remove the version pinning or pin to major version only.

If we settle on a supported version we can pin it then.